babyhandling.html

```diff
@@ -5,72 +5,188 @@
+        />
+      </svg>
+      <div
+        class="
+          flex flex-col flex-wrap
+          font-bold font-serif
+          text-2xl
+          tracking-wider
+          uppercase
+        "
+      >
+        <span>Kinästhetik </span>
+        <span>Baby Handling</span>
+      </div>
 
-      <h2>Baby richtig tragen</h2>
-      <p>
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quasi,
-        voluptatum! Excepturi nam molestias commodi qui eos dolorum blanditiis,
-        repellendus, exercitationem tempore officia quod cupiditate neque
-        deserunt repudiandae minus deleniti quam!
-      </p>
-      <img src="#" alt="" />
-
-      <h2>Baby richtig wickeln</h2>
-      <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat nisi
-        consequuntur aperiam porro, ea quo quidem nobis sapiente? Architecto
-        harum obcaecati iste doloribus aliquid, tempore necessitatibus.
-        Similique placeat quis sed!
-      </p>
-      <img src="#" alt="" />
+      <!--WILL USE MORE ACCESSIBLE FONTS IN MAIN-->
+    </header>
+    <main class="space-y-3 m-3 p-4">
+      <div class="p-6 font-mono text-center">
+        <h1 class="text-5xl text-blue-900" id="top">Baby Handling</h1>
+        <p class="text-blue-800 p-3">
+          So kannst du die Entwicklung deines Zwerges aktiv unterstützen!
+        </p>
+        <img
+          class="rounded-lg pb-8"
+          src="./h1baby.jpg"
+          alt="Baby feet in a soft white blanket"
+        />
+      </div>
+      <div class="space-y-3">
+        <h2 class="text-blue-800 text-3xl font-semibold">
+          Was ist Baby Handling?
+        </h2>
+        <p class="font-sans text-gray-800">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ad libero
+          saepe atque, reiciendis fuga, ratione a omnis iusto nam quasi,
+          consectetur facilis voluptatum quas ullam repudiandae assumenda
+          tempore animi nemo? Lorem ipsum dolor sit amet, consectetur
+          adipisicing elit. Ab culpa sint tenetur aut, temporibus cumque
+          dignissimos voluptatum ipsam, iste vitae quis quaerat totam deserunt
+          harum repudiandae excepturi beatae. Repellat, cupiditate.
+        </p>
+        <p class="font-semibold text-blue-800 text-lg">
+          Hier zeige ich dir wie du dein Baby:
+        </p>
+        <nav class="underline text-blue-900 text-opacity-75">
+          <ul class="list-disc list-inside">
+            <li><a href="#here1">Richtig hochnimmst und wieder hinlegst</a></li>
+            <li><a href="#here2">Richtig trägst</a></li>
+            <li><a href="#here3">Richtig wickelst</a></li>
+          </ul>
+        </nav>
```

Comment:
"As the <main> element is supposed to be used for the main, unique content of your site, your <nav> should not be inside it if it is navigation for your entire site." https://stackoverflow.com/questions/22549766/should-a-nav-tag-be-outside-the-main-tag
Ich glaube das nav Element ist eher für den Header gedacht, obwohl es technisch so geht. Ist eine gute Frage /Disskussion für morgen :)